README.md

```diff
@@ -23,6 +23,24 @@ def deps do
 end
 ```
 
+You'll also need to add `:artifact` to your list of applications, again in `mix.exs`, something like:
+
+```elixir
+def application do
+  [
+    applications: [
+    	:artifact
+    ]
+  ]
+end
+```
+
+It's also recommended to install goon (which ```artifact```/```porcelain``` uses) download the package for your system [here](https://github.com/alco/goon/releases) and unzip the contents somewhere in your path, for example:
+
+```bash
+sudo tar zxf goon_darwin_amd64.tar.gz -C /usr/local/bin/
```

Comment:
Let's add `$` here to differentiate this CLI example from code examples in the README:

```bash
$ sudo tar zxf goon_darwin_amd64.tar.gz -C /usr/local/bin/
```